src/config.rs

```diff
@@ -27,12 +27,20 @@ pub struct ServerConfig {
 #[derive(Deserialize, Debug)]
 pub struct RpcMethod {
     pub method: String,
+    pub params: Vec<MethodParam>,
```

Comment:
```suggestion
    #[serde(default)]
    pub params: Vec<MethodParam>,
```

I guess we can make this default so we don't need to define empty params